Controllers/PostController.cs

```diff
@@ -165,6 +177,18 @@ public IActionResult EditPost(PutPostByMeDTO puttedPost, int id)
                 _dbContext.PostTags.Add(postTag);
             }
 
+            if (puttedPost.FormFile != null)
+            {
+                byte[] file;
+                using (var memoryStream = new MemoryStream())
+                {
+                    puttedPost.FormFile.CopyTo(memoryStream);
+                    file = memoryStream.ToArray();
+                }
+            }
+
+            post.ImageBlob = file;
```

Comment:
does `post.ImageBlob = file; ` need to be moved into the if statement above it?  Getting an error that says `file` is out of context.

Reply:
yep, my bad